apps/vaporgui/VSection.cpp

```diff
@@ -75,3 +99,28 @@ void VSection::SettingsMenuButton::paintEvent(QPaintEvent *event)
     option.features = QStyleOptionToolButton::None;
     p.drawComplexControl(QStyle::CC_ToolButton, option);
 }
+
+VSection::ExpandSectionButton::ExpandSectionButton()
+{
+    setIcon(QIcon(QString::fromStdString(Wasp::GetSharePath("images/expandSection.png"))));
+    setIconSize(QSize(18, 18));
+    setCursor(QCursor(Qt::PointingHandCursor));
+    setPopupMode(QToolButton::InstantPopup);
+
+    setStyleSheet("border: none;"
+                  "background-color: none;"
+                  "padding: 0px;");
+}
+
+void VSection::ExpandSectionButton::paintEvent(QPaintEvent *event)
+{
+    // This function is overridden to prevent Qt from drawing its own dropdown arrow
+    QStylePainter p(this);
+
+    QStyleOptionToolButton option;
+    initStyleOption(&option);
+    option.subControls = QStyle::SC_ToolButton;
+    option.features = QStyleOptionToolButton::None;
+    p.drawComplexControl(QStyle::CC_ToolButton, option);
+}
+
```

Comment:
`ExpandSectionButton` duplicates the code of `SettingsMenuButton` apart from changing the icon path.